curp/src/server/raw_curp/log.rs

```diff
@@ -52,14 +57,27 @@ impl<C: 'static + Command> Log<C> {
     pub(super) fn new(
         log_tx: mpsc::UnboundedSender<LogEntry<C>>,
         entries: Vec<LogEntry<C>>,
+        batch_limit: u64,
     ) -> Self {
+        let mut batch_index = vec![0u64];
+        for entry in &entries {
+            #[allow(clippy::expect_used)]
```

Comment:
Add a comment to explain why. For instance, "it's in the initialization stage, panic doesn't affect much".

Reply:
Fixed